src/content/platform/cron-triggers/index.md

```diff
@@ -6,7 +6,7 @@ order: 10
 
 ## Background
 
-Cron Triggers allow users to map a cron expression to a Worker script using a [ScheduledEvent](/runtime-apis/scheduled-event) listener that enables Workers to be executed on a schedule. Cron Triggers are ideal for running periodic jobs for maintenance or calling third-party APIs to collect up-to-date data.
+Cron Triggers allow users to map a cron expression to a Worker script using a [ScheduledEvent](/runtime-apis/scheduled-event) listener that enables Workers to be executed on a schedule. Cron Triggers are ideal for running periodic jobs for maintenance or calling third-party APIs to collect up-to-date data. Cron Triggers are run on underutilized machines to make the best use of our capacity and route traffic efficiently.
```

Comment:
Nit: I would reframe to "Workers scheduled by Cron Triggers will on underutilized machines to make the best use of our capacity and route traffic efficiently"